arches_controlled_lists/src/arches_controlled_lists/components/tree/AddDeleteControls.vue

```diff
@@ -140,6 +140,13 @@ const toDelete = computed(() => {
         .map(([k]) => k);
 });
 
+function parseSingleDetail(error: unknown) {
+    if (!(error instanceof Error)) {
+        return undefined;
+    }
+    return error.message.split("\n").slice(0)[0];
```

Comment:
can the message this is receiving be refactored to clean this up?

Reply:
I also struggled with this one: the error reports every single node that is still using a list you attempted to delete, which could be dozens. That seems like good & correct information for an error to respond with (?) but overkill for a toast message, so it seemed like the toast was the place to chop. I guess we could alter the error response to add an arbitrary limit of some kind, but that seems too opinionated? WDYT?

Reply:
I'm happy to address this in a follow-up if you'd like to chat more -- but figure now's a good time to pull in the branch to unblock styling updates.